serendipity_plugin_shoutbox/serendipity_plugin_shoutbox.php

```diff
@@ -198,19 +188,19 @@ function generate_content(&$title)
            <input name='submit' type='submit' value='<?php echo PLUGIN_SHOUTBOX_SUBMIT; ?>' />
        </form>
 <?php
-        $q = 'SELECT    s.body              AS comment,
-                        s.timestamp         AS stamp,
-                        s.id                AS comment_id
-                FROM    '.$serendipity['dbPrefix'].'shoutbox AS s
-            ORDER BY    s.timestamp DESC
-               LIMIT ' . $max_entries;
+        $q = "SELECT body      AS comment,
+                     timestamp AS stamp,
+                     id        AS comment_id
+                FROM ${serendipity['dbPrefix']}shoutbox
+            ORDER BY timestamp DESC
+               LIMIT $max_entries";
 ?>
 <div class="serendipity_shoutbox">
 <?php
         $sql = serendipity_db_query($q);
         if ($sql && is_array($sql)) {
             foreach($sql AS $key => $row) {
-                $comments = wordwrap(strip_tags($row['comment']), $max_chars, '@@@', 1);
+                $comments = serendipity_specialchars(wordwrap($row['comment'], $max_chars, '@@@', 1));
```

Comment:
Just a note: `serendipity_specialchars` was introduced to fix a regression/design issue with charset support in PHP 5.4 and 5.5. It's something we could remove by now -> it is okay to call the PHP `htmlspecialchars` directly, if prefered,